### PR TITLE
simulators/ethereum/eth: add eth protocol test suite

### PR DESCRIPTION
**Pull request by renaynay**
This PR adds the eth protocol test suite to hive. 

Run it with the following cmd:
```bash 
./hive --sim ethereum/eth --sim.rootcontext --client <client>_<branch> --docker-noshell
```

### CODE REVIEW

**Comment by renaynay** [comment]
- geth working
- besu requests headers and disconnects
- nethermind docker container doesn't build
- openethereum needs protocol version 64
- trinity doesn't build

**Comment by renaynay** [comment]
update: 

- besu working
- openethereum has a bug? (wrong forkID in status and immediately sends PING upon fresh peer connection)
- nethermind can't import chain from chain.rlp file
- parity EOF issue on connecting
- trinity cannot import chain, also for some reason does not support eth 64 (but supports eth63 and eth65)

File: simulators/ethereum/eth/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This simulation runs various graphql tests
```
Comment (holiman) [inline simulators/ethereum/eth/Dockerfile:1]: Does it?

**Review by holiman — COMMENTED** [review]
Minor nitpick, otherwise LGTM

File: simulators/ethereum/eth/Dockerfile (hunk 1 above)
Comment (renaynay) [reply simulators/ethereum/eth/Dockerfile:1]: No it does not :) you caught me copy pasting.

**Pull request merged**